bundles/org.openhab.binding.dbquery/src/main/java/org/openhab/binding/dbquery/action/DBQueryActions.java

```diff
@@ -101,6 +104,19 @@ public ActionQueryResult getLastQueryResult() {
         }
     }
 
+    public static ActionQueryResult executeQuery(ThingActions actions, String query,
+            Map<String, @Nullable Object> parameters, int timeoutInSeconds) {
+        return ((DBQueryActions) actions).executeQuery(query, parameters, timeoutInSeconds);
+    }
+
+    public static void setQueryParameters(ThingActions actions, Map<String, @Nullable Object> parameters) {
+        ((DBQueryActions) actions).setQueryParameters(parameters);
+    }
+
+    public static ActionQueryResult getLastQueryResult(ThingActions actions) {
+        return ((DBQueryActions) actions).getLastQueryResult();
+    }
+
```

Comment:
What is the use of these three public methods in relation to the annotations and this PR?

Reply:
I added them because they were forgotten when these actions were implemented.

Reply:
I'll merge it, but i also like to know how this works, so hope you are willing to give me some insights.

These three methods seem 'duplicates' or overloads of the ones with annotations. Only with an extra parameter: `ThingActions actions`, but also lack the annotations.

Why do we need two sets? one for scripts and one for REST API?

Reply:
Answer my own question:
For reference: https://www.openhab.org/docs/developer/bindings/#actions-bound-to-a-thing

_Each member method also requires a static method with the same name. This is to support the old DSL rules engine and make the action available there._